typescript/create-onchain-agent/templates/next/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "next",
+  "name": "next-template",
```

Comment:
needed to rename this otherwise it would conflict with the actual `next` npm package